inlineplz/linters/__init__.py

```diff
@@ -277,13 +277,20 @@ def run_command(command, log_on_fail=False, log_all=False):
     shell = False
     if os.name == 'nt':
         shell = True
+    local_env = {}
+    safe_envvars = [
+        b'PATH', b'PYTHONPATH', b'GOPATH', b'SystemRoot',
```

Comment:
the idea here is to reduce risk when running potentially untrusted subprocesses by providing the absolute minimum necessary environment variables. i'm also going to refactor the way the token is being passed in the bot to make this safer.